lib/util/http_utils.dart

```diff
@@ -192,13 +192,22 @@ class HttpUtils {
 
 /// a wrapper class around the http Response
 class Response {
+  bool isLoading = false;
+  bool isSuccess = false;
```

Comment:
expose only getters so people don't change it outside the updateState()

Reply:
Done